gaeguli/pipeline.c

```diff
@@ -44,7 +45,9 @@ struct _GaeguliPipeline
   guint fps;
 
   GHashTable *targets;
+  GHashTable *rec_targets;
```

Comment:
Can't we handle recording pipelines in a general hashtable rather than defining feature specific container?